dt-assets/js/details.js

```diff
@@ -94,6 +95,19 @@ jQuery(document).ready(function($) {
     rest_api.update_post(post_type, post_id, { [fieldKey]: fieldValues }).then((newPost)=>{
       $(`#${fieldKey}-spinner`).removeClass('active')
       post = newPost
+
+      // Safeguard against immediate update failures.
+      if ( post && post[fieldKey] && post[fieldKey].length > 0 ) {
+        let updated_values = post[fieldKey].filter((option) => {
+          return (option['type'] === type) && (option['value'] === value);
+        });
+
+        // Assuming no duplicate values; process first and only filtered hit.
```

Comment:
@kodinkat Is this: add missing keys to the the new link field inputs?

Reply:
Flow updates the meta-id of the link input which generated the event, with newly created meta-id. This ensures any immediate updates, are assigned to correct link input and not to a new/duplicated input field.